com.unity.render-pipelines.universal/ShaderLibrary/Debug/DebugViewEnums.cs

```diff
@@ -113,6 +113,19 @@ public enum DebugSceneOverrideMode
         ShadedWireframe,
     }
 
+    /// <summary>
+    /// Debug mode of the overdraw
+    /// </summary>
+    public enum DebugOverdrawMode
+    {
+        /// <summary>Debug overdraw of opaque only.</summary>
+        Opaque,
+        /// <summary>Debug overdraw of transparent only.</summary>
+        Transparent,
+        /// <summary>Debug overdraw of everything only.</summary>
+        All
+    }
```

Comment:
The convention for other debug modes is to have the first entry here be "None". This would tell whether the mode is enabled, whereas now you kept the separate boolean `overdraw` for this purpose. I would prefer to do it this way instead to be consistent with other debug modes. The UI will be a little simpler too (you would display the overdraw dropdown always, and the `isHidden` condition for the `max cost` sub-option would be `overdrawMode == DebugOverdrawMode.None`.